packages/components/src/internal/components/user/UserProfile.spec.tsx

```diff
@@ -44,7 +42,7 @@ describe('<UserProfile/>', () => {
         expect(wrapper.find('.user-text-link')).toHaveLength(0);
         expect(wrapper.find(TextInput)).toHaveLength(5);
         expect(wrapper.find('input').findWhere(input => input.prop('disabled'))).toHaveLength(1); // email disabled
-        expect(wrapper.find(Button)).toHaveLength(2);
+        expect(wrapper.find(Button)).toHaveLength(1);
```

Comment:
which button went away? was it the Cancel button, if so was that intentional?

Reply:
The cancel button, and yes it was intentional. I ran this change past Hannah and Bernie, both agreed it wasn't necessary for this page.